libafl_sugar/src/inmemory.rs

```diff
@@ -137,7 +137,7 @@ where
 
         let shmem_provider = StdShMemProvider::new().expect("Failed to init shared memory");
 
-        let monitor = MultiMonitor::new(|s| log::info!("{s}"));
+        let monitor = MultiMonitor::new(|s| println!("{s}"));
```

Comment:
Instead of using `println` here, please use a logger such as `env_logger` that prints.

Reply:
That is a doubt I have. Following [this comment](https://github.com/AFLplusplus/LibAFL/issues/127#issuecomment-848695351), the output from `MultiMonitor` should be treated as a _"output for the user during fuzzing"_ and be independent of the log level, shouldn't? Using a logger like `env_logger` wouldn't mix these concerns?

Reply:
> That is a doubt I have. Following [this comment](https://github.com/AFLplusplus/LibAFL/issues/127#issuecomment-848695351), the output from `MultiMonitor` should be treated as a _"output for the user during fuzzing"_ and be independent of the log level, shouldn't? Using a logger like `env_logger` wouldn't mix these concerns?

Yes, logger are for library code, fuzzers are frontends, println is ok